src/dftbp/dftb/periodic.F90

```diff
@@ -430,15 +452,15 @@ subroutine updateNeighbourList(coord, img2CentCell, iCellVec, neigh, nAllAtom, c
 
     real(dp) :: dist2
     real(dp) :: rCell(3), rr(3)
-    integer :: ii, iAtom1, oldIAtom1, iAtom2
+    integer :: ii, iAtom1, oldIAtom1, iAtom2, startAtom, endAtom
     integer :: nn1, iAtom2End
-    logical :: symm
-    integer, allocatable :: indx(:)
+    logical :: symm, parallel, parallelSetupError
```

Comment:
Could you switch to the more recent variable convention in use elsewhere in the code (and yes, symm doesn't use this unfortunately) of isParallel and isParallelSetupError ?